app/views/applications/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<h1><%= @app.name %></h1>
+<li> Status:   <%= "#{@app.status}"%></li>
+<li> Street:   <%= "#{@app.address}"%></li>
+<li> City:     <%= "#{@app.city}"%></li>
+<li> State:    <%= "#{@app.state}"%></li>
+<li> Zip Code: <%= "#{@app.zipcode}"%></li>
+<li> Description: <%= "#{@app.description}"%></li>
+
+<h3>Links To Pets On Application</h3>
+
+<% @app.pets.each do |pet| %>
+    <li><%= link_to "#{pet.name}", "/pets/#{pet.id}" %></li>
+<% end %>
+
+<h3>Add a Pet to this Application</h3>
+
+<%= form_with url: "/applications/#{@app.id}", method: :get, local: true do |form| %>
+  <%= form.label :search_name, "Search for a Pet:" %>
+  <%= form.text_field :search_name %> 
+  <%= form.submit "Submit" %>
+<% end %>
+
+<% if @pet_search_result %>
+  <% @pet_search_result.each do |pet| %>
+    <li><%= pet.name %></li>
+  <% end %>
+<% end %>
```

Comment:
Again, if you are going to use `<li>`'s then they must be enclosed in a list, either a `<ul>` or `<ol>`.